src/openforms/registrations/admin.py

```diff
@@ -12,3 +19,22 @@ def formfield_for_dbfield(self, db_field, request, **kwargs):
             return field
 
         return super().formfield_for_dbfield(db_field, request, **kwargs)
+
+
+class TestPluginAdmin(admin.ModelAdmin):
```

Comment:
This admin model is more something for the config app. Move it there. Eventually, I want to test ALL plugins and not just the registration pluguins.

Also, there is no need to use the `admin.ModelAdmin` at all, since we don't have any (real) models to use it with. We can just add a TemplateView like done here (done with FormView but we don't need a form): https://github.com/open-formulieren/open-forms/blob/master/src/openforms/urls.py#L32-L36

Reply:
view added as TemplateView